native/core/src/parquet/parquet_support.rs

```diff
@@ -60,9 +60,6 @@ pub struct SparkParquetOptions {
     pub allow_incompat: bool,
     /// Support casting unsigned ints to signed ints (used by Parquet SchemaAdapter)
     pub allow_cast_unsigned_ints: bool,
-    /// We also use the cast logic for adapting Parquet schemas, so this flag is used
-    /// for that use case
-    pub is_adapting_schema: bool,
```

Comment:
This is dead code from when we used the cast logic (and `CastOptions`) to handle Parquet type conversion.